commands/release.js

```diff
@@ -44,7 +44,7 @@ function release(context) {
       var imageId = docker.ensureStartImage(context.cwd);
       if (!imageId) return Promise.reject(new Error('Unable to find a start image'));
 
-      var containerId = child.execSync(`docker run -d ${imageId} tar cfvz /tmp/slug.tgz -C / --exclude=.git --exclude=.heroku ./app`, {
```

Comment:
I think we need to keep ./app, right, but just remove the empty --exclude from the end of the last version?